cacher/cacher.go

```diff
@@ -18,13 +18,58 @@ import (
 	"hash/fnv"
 	"io/ioutil"
 	"net/http"
+	"net/url"
 	"os"
 	"path"
+	"strconv"
 	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/travelaudience/go-promhttp"
 
 	"github.com/relex/gotils/logger"
 )
 
+var (
+	totalCacheRequests = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cacher_cache_requests_total",
+			Help: "The total number of requests which were routed to the cache.",
+		},
+		[]string{"result", "path"})
```

Comment:
Would it make sense to change`path` to `url` since the `cacher_requests_total` metric has such a label?